src/charm.py

```diff
@@ -340,12 +340,19 @@ def _render_conf_file(self) -> str:
 
         default_schema_id, schemas = self._get_identity_schema_config()
         login_ui_url = self._get_login_ui_endpoint_info("login_url")
+
+        allowed_return_urls = []
+        if self._public_url:
+            allowed_return_urls.append(domain_url(self._public_url))
+        if login_ui_url:
+            allowed_return_urls.append(login_ui_url)
```

Comment:
I don't think this is needed anymore (the login UI is in the same domain)

Reply:
true, thanks for the catch!